### PR TITLE
BandolierForNPC.esp added

### PR DESCRIPTION
**Pull request by coco9515**
I added my mod : [BandolierForNPC.esp](https://www.nexusmods.com/skyrimspecialedition/mods/19093) - and 4 others .esp to the list.

All of them required Dr_Bandolier.esp.
BandolierForNPC.esp is the main .esp but :
 - 3 of them are standalone compatibility patch, all the required .esp has been placed.
 - The last one is an optional add-on to my mod or Dr_Bandolier.esp.

### CODE REVIEW

**Comment by Ortham** [comment]
If you've got a plugin in `req`, you don't need it in `after` too. If you remove the redundant entries, everything else looks fine.

**Comment by coco9515** [comment]
Okay, I updated. Is everything okay now ?

File: masterlist.yaml [hunk 1]
```diff
@@ -5460,3 +5484,33 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - 'Cloaks.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: File masters automatically have a required entry, there is no need to add these as required if they're in the `esp` header.

File: masterlist.yaml [hunk 2]
```diff
@@ -5460,3 +5484,33 @@ plugins:
...
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Cloaks of Skyrim Patch
+    tag:
+      - Invent
+  - name: 'BandolierForNPC - Winter is Coming Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Same as above.

File: masterlist.yaml [hunk 3]
```diff
@@ -5460,3 +5484,33 @@ plugins:
...
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Winter is Coming Patch
+    tag:
+      - Invent
+  - name: 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Same as above.

File: masterlist.yaml [hunk 4]
```diff
@@ -3857,6 +3857,30 @@ plugins:
     clean:
       - crc: 0x65AFCE11 # v4.06SE
         util: 'SSEEdit v3.2.2'
+  - name: 'BandolierForNPC.esp'
+    req:
+      - 'Dr_Bandolier.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Also not needed.

File: masterlist.yaml [hunk 5]
```diff
@@ -3857,6 +3857,30 @@ plugins:
...
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC
+    tag:
+      - Invent
+  - name: 'BandolierForNPC - Realistic Enchantements.esp'
+    req:
+      - 'Dr_Bandolier.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Also not needed.

**Review by MacSplody — CHANGES_REQUESTED** [review]
>If manually installing - choose only one

if only one of the below plugins can be installed then an `inc:` key should be used to ensure this.
`BandolierForNPC.esp`
`BandolierForNPC - Cloaks of Skyrim Patch`
`BandolierForNPC - Winter is Coming Patch`
`BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch`

so for example:

```yaml

  - name: 'BandolierForNPC.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093/'
        name: 'Bandolier - Bags and Pouches for NPC on Nexus Mods'
    inc:
      - 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
      - 'BandolierForNPC - Winter is Coming Patch.esp'
      - 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
    tag:
      - Names
      - Stats
```

**Comment by MacSplody** [comment]
Also I usually mark mods that have masters in the wrong order as requiring sorting.
But since this is your mod, I'll leave it up to you if you want to fix that.

`BandolierForNPC - Realistic Enchantements.esp`

image

File: masterlist.yaml [hunk 6]
```diff
@@ -3857,6 +3857,33 @@ plugins:
...
+    after:
+      - 'Dr_Bandolier.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: - 'Dr_Bandolier.esp' is not needed in `after` as it's a master.

File: masterlist.yaml [hunk 7]
```diff
@@ -5460,3 +5487,42 @@ plugins:
...
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
+    after:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
+      - '1nivWICSkyCloaksPatch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Winter is Coming and Cloaks of Skyrim Patch
```
Comment (MacSplody) [inline masterlist.yaml]: It's purely aesthetic but I usually keep `url` keys just under the `name` key for consistency.

```yaml

  - name: 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093/'
        name: 'Bandolier - Bags and Pouches for NPC on Nexus Mods'

```

**Review by MacSplody — CHANGES_REQUESTED** [review]
Okay so I need you to add `inc` entries for all standalone version so that only one version is used.

You can remove any Req & After entries where the plug-in is a master, as LOOT will handle requires and load after automatically for masters.

File: masterlist.yaml [hunk 8]
```diff
@@ -3857,6 +3857,30 @@ plugins:
...
       - crc: 0x65AFCE11 # v4.06SE
         util: 'SSEEdit v3.2.2'
+  - name: 'BandolierForNPC.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC
+    tag:
+      - Invent
```
Comment (MacSplody) [inline masterlist.yaml]: You don't need the `Invent` tag as that is for changes to `NPC_` or `CONT` Container records.

The following Bash tags can be added as well to all your plug-ins.
```yaml
    tag:
      - Names
      - Stats
```

**Comment by coco9515** [comment]
Ok, I think I did everything needed. Could you check ? Thank you for your time :)

**Review by MacSplody — APPROVED** [review]
Yeah looks good, Thanks for the contribution.

**Pull request merged**